docs/extensions/semgrep-intellij.md

```diff
@@ -0,0 +1,102 @@
+---
+slug: semgrep-intellij
+append_help_link: true
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Semgrep IntelliJ extension
+
+[Semgrep](https://semgrep.dev/) accelerates your security journey by swiftly scanning code and package dependencies for known issues, software vulnerabilities, and detected secrets with unparalleled efficiency. By default, the Semgrep IntelliJ extension scans code whenever you change or open files.
```

Comment:
```suggestion
[Semgrep](https://semgrep.dev/) swiftly scans code and package dependencies for known issues, software vulnerabilities, and detected secrets. Run Semgrep in your developer environment with the IntelliJ extension to catch code issues as you type. By default, the Semgrep IntelliJ extension scans code whenever you change or open files.
```
As per Chinmay's observation further down the doc, I've suggested some trims to the more markety bits and focused on the functionality.